build.gradle

```diff
@@ -1,11 +1,16 @@
 buildscript {
+    ext.kotlin_version = '1.1.60'
```

Comment:
1.2.0

Reply:
![screen shot 2017-12-01 at 12 56 29](https://user-images.githubusercontent.com/18715547/33481958-14f44a0a-d697-11e7-9fee-823025b1c732.png)

Reply:
kdyz to hodim na 1.2.0